source/retryable-reads/tests/gridfs-download-serverErrors.yml

```diff
@@ -0,0 +1,152 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "files"
+
+data:
+    files:
+        -
+            _id: { "$oid" : "000000000000000000000003" }
+            length: 2
+            chunkSize: 4
+            uploadDate: { "$date" : "1970-01-01T00:00:00.000Z" }
+            md5: "c700ed4fdb1d27055aa3faa2c2432283"
+            filename: "length-2"
+            contentType: "application/octet-stream"
+            aliases: []
+            metadata: {}
+    chunks:
+        - { _id : { "$oid" : "000000000000000000000003" }, files_id : { "$oid" : "000000000000000000000004" }, n : 0, data : { $hex : "11223344" } }
```

Comment:
Should your test README refer to the GridFS spec's test README for information about handling `$hex` when loading data fixtures?

Reply:
Second thought: since this spec has its own test runner I wonder if we can just use actual extended JSON for binary data instead of the test-specific `$hex` format. IIRC, the GridFS spec/tests were written before we had a standardized extended JSON format.